mail_menager.py

```diff
@@ -0,0 +1,53 @@
+from db_menager import DBManager
+from user_menager import UserManager
+
+
+class MailManager(DBManager):
+    def __init__(self) -> None:
+        self.connection = self.get_connection()
```

Comment:
maybe move it to DBManager because looks like copy-paste here and in the User manager 

+ rename files. make it snake_case but with the mAnager.